CMakeLists.txt

```diff
@@ -340,13 +340,6 @@ if(EXECUTORCH_BUILD_QNN)
   add_subdirectory(${CMAKE_CURRENT_SOURCE_DIR}/examples/qualcomm)
 endif()
 
-# Build Arm Baremetal backend
-option(EXECUTORCH_BUILD_ARM_BAREMETAL
```

Comment:
we want this switch and a way to build backend from the top level for the top level executor_runner target because examples is optional i.e. end-users may not need that, but this is to gate building the arm backend and should be same for all backends.

Reply:
@larryliu0820